Mathlib/Algebra/Associated.lean

```diff
@@ -266,7 +266,7 @@ theorem irreducible_isUnit_mul {a b : α} (h : IsUnit a) : Irreducible (a * b) 
 
 theorem irreducible_mul_units (a : αˣ) (b : α) : Irreducible (b * ↑a) ↔ Irreducible b := by
   simp only [irreducible_iff, Units.isUnit_mul_units, and_congr_right_iff]
-  refine' fun _ => ⟨fun h A B HAB => _, fun h A B HAB => _⟩
+ h A B HAB => _, fun h A B HAB => _⟩
```

Comment:
Something strange happens with indentation here.

Reply:
WIP...

Reply:
thanks, removed this spurious edit..